examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -191,17 +191,15 @@ class _ExpansionPanelsDemoState extends State<ExpasionPanelsDemo> {
 
           return new CollapsibleBody(
             margin: const EdgeInsets.symmetric(horizontal: 16.0),
-            child: new Input(
-              hintText: item.hint,
-              labelText: item.name,
-              value: new InputValue(
-                text: item.value
+            child: new Form(
+              child: new Input(
+                hintText: item.hint,
+                labelText: item.name,
+                value: new InputValue(text: item.value),
```

Comment:
I think this is an indication that the API is confusing. But the way it works is that when there's a formField, the `value` param is only used for the initial value. Subsequent values are taken from the Form.

What you suggest should work, however. I just think using Form looks simpler/cleaner.